src/pilot/templates/static-ip-environment.yaml

```diff
@@ -25,6 +25,13 @@ resource_registry:
   OS::TripleO::Compute::Ports::StorageMgmtPort: ./overcloud/network/ports/noop.yaml
   OS::TripleO::Compute::Ports::TenantPort: ./overcloud/network/ports/tenant_from_pool.yaml
 
+
```

Comment:
I think it will be confusing to leave the OS::TripleO::Compute entries in this file.  Can you remove them?  I think it should be safe to do.